chromium_src/chrome/browser/about_flags.cc

```diff
@@ -158,7 +159,11 @@ using ntp_background_images::features::kBraveNTPSuperReferralWallpaper;
     {"brave-permission-lifetime",                                          \
      flag_descriptions::kBravePermissionLifetimeName,                      \
      flag_descriptions::kBravePermissionLifetimeDescription, kOsAll,       \
-     FEATURE_VALUE_TYPE(permissions::features::kPermissionLifetime)},
+     FEATURE_VALUE_TYPE(permissions::features::kPermissionLifetime)},      \
+    {"brave-rewards-bitflyer",                                             \
```

Comment:
Should this be behind `#if BUILDFLAG(BRAVE_REWARDS_ENABLED)` flag?

Reply:
I believe that `BRAVE_REWARDS_ENABLED` hasn't been maintained correctly and is not working at the moment, and the plan is to remove it in a future PR. Given that, do you think we should put this behind a buildflag until then or leave this as-is?

Reply:
If we are going to remove it then probably no need to add it here. Thanks!